RecoParticleFlow/PFClusterTools/src/PFEnergyCalibration.cc

```diff
@@ -161,6 +161,59 @@ void PFEnergyCalibration::initializeCalibrationFunctions() {
   fdEtaEndcapEH->SetParameter(3, 1.0);
 }
 
+PFEnergyCalibration::CalibratedEndcapPFClusterEnergies PFEnergyCalibration::calibrateEndcapClusterEnergies(
+    reco::PFCluster const& eeCluster,
+    std::vector<reco::PFCluster const*> const& psClusterPointers,
+    ESChannelStatus const& channelStatus,
+    bool applyCrackCorrections) const {
+  double ps1_energy_sum = 0.;
+  double ps2_energy_sum = 0.;
+  double ePS1 = 0.;
+  double ePS2 = 0.;
+  int condP1 = 1;
```

Comment:
Maybe if you make these two some "bool" variable, everything gets more intuitive